Index: CISCO-WIRELESS-IF-MIB.my
===================================================================
diff -u -r1.1.1.1 CISCO-WIRELESS-IF-MIB.my
--- CISCO-WIRELESS-IF-MIB.my	1 Nov 2004 16:55:34 -0000	1.1.1.1
+++ CISCO-WIRELESS-IF-MIB.my	2 Nov 2004 22:57:20 -0000
@@ -247,7 +247,7 @@
 		FROM CISCO-TC;
 
 	ciscoWirelessIfMIB MODULE-IDENTITY
-           LAST-UPDATED "200025211500Z"
+           LAST-UPDATED "200004060000Z"
            ORGANIZATION "Cisco Systems Inc."
            CONTACT-INFO
 	   "	Cisco Systems
Index: DOCS-CABLE-DEVICE-TRAP-MIB.my
===================================================================
diff -u -r1.1.1.1 DOCS-CABLE-DEVICE-TRAP-MIB.my
--- DOCS-CABLE-DEVICE-TRAP-MIB.my	1 Nov 2004 16:55:34 -0000	1.1.1.1
+++ DOCS-CABLE-DEVICE-TRAP-MIB.my	2 Nov 2004 22:57:20 -0000
@@ -69,7 +69,7 @@
                   Docsis 2.0 RF MIB draft were added to the trap
                   definitions."
 
-            REVISION "000926000000Z"
+            REVISION "200212200000Z"
             DESCRIPTION
                 "The CABLE DEVICE TRAP MIB is an extension of the
                  CABLE DEVICE MIB defined in RFC2669. It defines various
