--- /home/max/mibs/hp/feb05/hpicfXrrp.mib	Tue Feb 15 14:40:24 2005
+++ hpicfxrrp.mib	Fri Dec  2 19:36:40 2005
@@ -10,6 +10,8 @@
             FROM Q-BRIDGE-MIB
         LastCreateTime
                 FROM RMON2-MIB
+        hpSwitch
+            FROM HP-ICF-OID
         TEXTUAL-CONVENTION, TruthValue, RowStatus, TimeStamp, MacAddress
             FROM SNMPv2-TC;
 
@@ -25,11 +27,11 @@
                       proprietory version of the Virtual Router Redundancy
                       Protocol (VRRP)."
 
-        REVISION      "200410200000Z  -- October 20, 2004"
+        REVISION      "200410200000Z" -- October 20, 2004
         DESCRIPTION   "Revision to add objects to support controlled 
                        failback feature for XRRP."
                        
-        REVISION      "200208190000Z  -- August 19, 2002"
+        REVISION      "200208190000Z" -- August 19, 2002
         DESCRIPTION   "Initial revision."
         
         ::= { hpSwitch 18 }
