diff -ru v2.orig/CISCO-DSL-PROVISION-MIB.my cisco-v2/CISCO-DSL-PROVISION-MIB.my
--- v2.orig/CISCO-DSL-PROVISION-MIB.my 2001-08-27 19:16:14.000000000 +0200
+++ cisco-v2/CISCO-DSL-PROVISION-MIB.my    2004-04-25 14:15:59.930927438 +0200
@@ -35,7 +35,7 @@
                 FROM CISCO-SMI;
 
 ciscoDslProvisionMIB MODULE-IDENTITY
-        LAST-UPDATED        "990618000000Z"
+        LAST-UPDATED        "9906180000Z"
         ORGANIZATION        "Cisco Systems, Inc."
         CONTACT-INFO        " Cisco Systems
                         Customer Service
diff -ru v2.orig/CISCO-ICSUDSU-MIB.my cisco-v2/CISCO-ICSUDSU-MIB.my
--- v2.orig/CISCO-ICSUDSU-MIB.my   1999-06-25 21:52:43.000000000 +0200
+++ cisco-v2/CISCO-ICSUDSU-MIB.my  2004-04-25 14:13:00.139588656 +0200
@@ -28,7 +28,7 @@
         ifIndex
                 FROM RFC1213-MIB
         dsx1ConfigEntry
-                FROM RFC1406-MIB
+                FROM DS1-MIB
         ciscoMgmt
                 FROM CISCO-SMI;
 
diff -ru v2.orig/CISCO-IDSL-LINE-MIB.my cisco-v2/CISCO-IDSL-LINE-MIB.my
--- v2.orig/CISCO-IDSL-LINE-MIB.my 2000-07-02 05:04:10.000000000 +0200
+++ cisco-v2/CISCO-IDSL-LINE-MIB.my    2004-04-25 14:20:46.736796346 +0200
@@ -26,7 +26,6 @@
     PerfCurrentCount,
     PerfIntervalCount                   FROM PerfHist-TC-MIB
     SnmpAdminString                     FROM SNMP-FRAMEWORK-MIB
-    AdslLineProfileName,
     adslLineConfProfileName             FROM ADSL-LINE-MIB
     ;
 
diff -ru v2.orig/CISCO-POP-MGMT-MIB.my cisco-v2/CISCO-POP-MGMT-MIB.my
--- v2.orig/CISCO-POP-MGMT-MIB.my  2001-05-04 18:53:16.000000000 +0200
+++ cisco-v2/CISCO-POP-MGMT-MIB.my 2004-04-25 14:07:13.031415438 +0200
@@ -32,7 +32,7 @@
                FROM CISCO-TC
         dsx1LineStatus,
         dsx1LineIndex
-               FROM RFC1406-MIB
+               FROM DS1-MIB
         ciscoExperiment
                 FROM CISCO-SMI;
 
diff -ru v2.orig/CISCO-SDSL-LINE-MIB.my cisco-v2/CISCO-SDSL-LINE-MIB.my
--- v2.orig/CISCO-SDSL-LINE-MIB.my 2001-12-04 05:18:18.000000000 +0100
+++ cisco-v2/CISCO-SDSL-LINE-MIB.my    2004-04-25 14:19:30.836358131 +0200
@@ -22,7 +22,6 @@
     MODULE-COMPLIANCE,
     OBJECT-GROUP                        FROM SNMPv2-CONF
     ciscoMgmt                           FROM CISCO-SMI
-    AdslLineProfileName,
     adslLineConfProfileName,
     adslLineAlarmConfProfileName        FROM ADSL-LINE-MIB
     ifIndex                             FROM IF-MIB
diff -ru v2.orig/CISCO-VPDN-MGMT-EXT-MIB.my cisco-v2/CISCO-VPDN-MGMT-EXT-MIB.my
--- v2.orig/CISCO-VPDN-MGMT-EXT-MIB.my 2001-08-24 22:46:35.000000000 +0200
+++ cisco-v2/CISCO-VPDN-MGMT-EXT-MIB.my    2004-04-25 13:59:31.369718823 +0200
@@ -27,7 +27,7 @@
         ciscoExperiment
                 FROM CISCO-SMI;
 ciscoVpdnMgmtExtMIB MODULE-IDENTITY
-        LAST-UPDATED        "990414000000Z"
+        LAST-UPDATED        "9904140000Z"
         ORGANIZATION        "Cisco Systems, Inc."
         CONTACT-INFO        "Cisco Systems
                              Customer Service
diff -ru v2.orig/CISCO-WIRELESS-IF-MIB.my cisco-v2/CISCO-WIRELESS-IF-MIB.my
--- v2.orig/CISCO-WIRELESS-IF-MIB.my   2000-04-06 22:10:14.000000000 +0200
+++ cisco-v2/CISCO-WIRELESS-IF-MIB.my  2004-04-25 13:55:26.138695574 +0200
@@ -247,7 +247,7 @@
       FROM CISCO-TC;
 
   ciscoWirelessIfMIB MODULE-IDENTITY
-           LAST-UPDATED "200025211500Z"
+           LAST-UPDATED "200205211500Z"
            ORGANIZATION "Cisco Systems Inc."
            CONTACT-INFO
      "    Cisco Systems
