Only in hp: hp.bad
diff -ur hp.orig/hpicfbasic.mib hp/hpicfbasic.mib
--- hp.orig/hpicfbasic.mib	2009-06-05 16:25:57.000000000 -0700
+++ hp/hpicfbasic.mib	2009-06-05 16:35:51.000000000 -0700
@@ -1446,9 +1446,9 @@
                         other than responses to network management
                         requests (e.g. repeaters and switches)."
 
-            GROUP       hpicfBasicSnmpNotifyLogFilterTable
-            DESCRIPTION "This group is required for devices that support
-                         SNMPv3."
+--            GROUP       hpicfBasicSnmpNotifyLogFilterTable
+--             DESCRIPTION "This group is required for devices that support
+--                         SNMPv3."
 
             OBJECT      hpicfIpConfigPingRouter
             MIN-ACCESS  read-only
diff -ur hp.orig/hpicfbridge.mib hp/hpicfbridge.mib
--- hp.orig/hpicfbridge.mib	2009-06-05 16:25:57.000000000 -0700
+++ hp/hpicfbridge.mib	2009-06-05 17:10:30.000000000 -0700
@@ -15,7 +15,7 @@
             FROM HP-ICF-OID
         ConfigStatus
             FROM HP-ICF-TC
-        hpSwitchStpProtocolVersion, hpSwitchStpPortEntry
+        hpSwitchStpPortEntry
             FROM CONFIG-MIB
         ifIndex
             FROM IF-MIB
diff -ur hp.orig/hpicfipaddress.mib hp/hpicfipaddress.mib
--- hp.orig/hpicfipaddress.mib	2009-06-05 16:25:57.000000000 -0700
+++ hp/hpicfipaddress.mib	2009-06-05 16:43:38.000000000 -0700
@@ -10,7 +10,9 @@
         InetAddress, InetAddressType, InetAddressPrefixLength, InetAddressIPv4, 
         InetPortNumber
             FROM INET-ADDRESS-MIB
-        ifIndex,ipNetToPhysicalEntry
+        ipNetToPhysicalEntry
+            FROM IP-MIB
+        ifIndex
             FROM IF-MIB
         hpicfCommon
             FROM HP-ICF-OID
diff -ur hp.orig/hpicfl3macconfig.mib hp/hpicfl3macconfig.mib
--- hp.orig/hpicfl3macconfig.mib	2009-06-05 16:25:57.000000000 -0700
+++ hp/hpicfl3macconfig.mib	2009-06-05 17:12:28.000000000 -0700
@@ -1,6 +1,8 @@
 HP-ICF-L3MAC-MIB DEFINITIONS ::= BEGIN
 
     IMPORTS
+        hpSwitch            
+            FROM HP-ICF-OID            
         OBJECT-TYPE, MODULE-IDENTITY
             FROM SNMPv2-SMI
         RowStatus,PhysAddress
diff -ur hp.orig/hpicfproviderbridge.mib hp/hpicfproviderbridge.mib
--- hp.orig/hpicfproviderbridge.mib	2009-06-05 16:25:57.000000000 -0700
+++ hp/hpicfproviderbridge.mib	2009-06-05 17:13:13.000000000 -0700
@@ -1,6 +1,8 @@
 HP-ICF-PROVIDER-BRIDGE DEFINITIONS ::= BEGIN
 
     IMPORTS
+        hpSwitch            
+            FROM HP-ICF-OID            
         OBJECT-TYPE, MODULE-IDENTITY
             FROM SNMPv2-SMI
         TruthValue
diff -ur hp.orig/hpicfusrauth.mib hp/hpicfusrauth.mib
--- hp.orig/hpicfusrauth.mib	2009-06-05 16:25:57.000000000 -0700
+++ hp/hpicfusrauth.mib	2009-06-05 16:48:14.000000000 -0700
@@ -11,9 +11,11 @@
             FROM SNMP-FRAMEWORK-MIB            
         OBJECT-GROUP, MODULE-COMPLIANCE, NOTIFICATION-GROUP            
             FROM SNMPv2-CONF            
-        IpAddress, RowStatus, Integer32, Unsigned32, OBJECT-TYPE, 
+        IpAddress, Integer32, Unsigned32, OBJECT-TYPE, 
         MODULE-IDENTITY, NOTIFICATION-TYPE            
             FROM SNMPv2-SMI            
+        RowStatus
+            FROM SNMPv2-TC
         TruthValue, MacAddress            
             FROM SNMPv2-TC
         InetAddressType, InetAddress
