--- rapid_city.mib	2007-10-17 22:02:23.000000000 -0400
+++ ../../rapid_city.mib	2007-10-17 21:54:06.000000000 -0400
@@ -166,6 +166,10 @@
 rcA1612                 OBJECT IDENTIFIER ::= { rapidCity 44 }
 rcA1624                 OBJECT IDENTIFIER ::= { rapidCity 45 }
 rcRBS2402               OBJECT IDENTIFIER ::= { rapidCity 46 }
+rcAtr8310               OBJECT IDENTIFIER ::= { rapidCity 47 }
+rcAtr8306               OBJECT IDENTIFIER ::= { rapidCity 48 }
+rcAtr8010               OBJECT IDENTIFIER ::= { rapidCity 49 }
+rcAtr8006               OBJECT IDENTIFIER ::= { rapidCity 50 }
 -- From rapidCity 100 to rapidCity 200 is reserved for OM chassises
 
 rcSystem  		OBJECT IDENTIFIER ::= { rcMgmt 1 }
